src/XMonad/Main.hs

```diff
@@ -186,24 +193,12 @@ xmonadNoargs initxmc serializedWinset serializedExtstate = do
                  return (fromMaybe fbc_ v)
 
     hSetBuffering stdout NoBuffering
+    xinesc <- getCleanedScreenInfo dpy
```

Comment:
What was the reason for moving this line down?  I don't really know what it does so I'm just curious.